table/server/master/src/main/java/alluxio/master/table/Table.java

```diff
@@ -86,6 +86,8 @@ private Table(Database database, UdbTable udbTable, @Nullable Table previousTabl
           previousTable.mPartitionScheme.getPartitions().stream()
               .collect(Collectors.toMap(Partition::getSpec, Function.identity()));
       for (UdbPartition udbPartition : udbTable.getPartitions()) {
+        LOG.debug("Get partition: {}, from table {} Database {}.",
+            udbPartition.toString(), database.getName(), mName);
         Partition newPartition = existingPartitions.get(udbPartition.getSpec());
```

Comment:
Is `udbPartition.getSpec()` fully captured in `udbPartition.toString()`? We use the spec to identify the partition, so that would be important information to log.

Reply:
Added the Spce in `toString()`.